test/__bootstrap__/3rdparty/expect.assertions.js

```diff
@@ -0,0 +1,40 @@
+// Based on jasmine-expect-count.
+// Copyright (c) 2017 Tony Brix. Released under MIT license:
+// https://github.com/UziTech/jasmine-expect-count
+
+EXPECT_PATCHING_SCOPE: {
+    if (!globalThis.jasmine) {
+        throw new Error("jasmine must be loaded before patching in expect.assertions(x)");
+    }
+
+    let actualExpects = 0;
+    let expectedExpects = null;
+
+    beforeEach(() => {
+        actualExpects = 0;
+        expectedExpects = null;
+    });
+
+    const { expect: originalExpect } = globalThis;
+
+    globalThis.expect = function(...args) {
+        actualExpects++;
+        return Reflect.apply(originalExpect, this, args)
```

Comment:
Style nit: Missing trailing semicolon. ☝️

Reply:
Glad I did this https://github.com/salesforce/near-membrane/pull/156